server.py

```diff
@@ -50,8 +50,168 @@ def return_items_needed_by_given_foodbank (name_of_foodbank): # Give a foodbank
 
 class index:
 	def GET(self):
-		return "Hello! This is an API that gives information on nearby foodbanks and the items they need. Check out the documentation, when it exists..."
+		return "Hello! This is an API that gives information on nearby food banks and the items they need. Check out the documentation, when it exists..."
 
+class documentation:
+	def GET(self):
+		return 
+	<!DOCTYPE html>
+<html>
+<body>
+
+<h1 id="Foodbank API">Foodbank API</h1>
+<p>Contains information about individual food items needed by a Trussell Trust food bank specified by name or location. 
+
+<h2 id="Base URL">Base URL</h2>
+<p>http://whatfoodbanksneed.org.uk</p>
+
+<h3 id="Endpoint 1">Endpoint 1</h3>
+<p>GET /Individual_foodbank_information</p>
+
+<h4 id="Description">Description</h4>
+<p>This returns the food bank name and the food items needed for the named food bank.</p>
+
+<h4 id="Query string parameters">Query string parameters</h4>
+
+<head>
+<style>
+table, th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+</head>
+<body>
+
+<table style="width:100%">
+  <tr>
+    <th>Query string parameter</th>
+    <th>Required/optional</th> 
+    <th>Description</th>
+    <th>Type</th>
+  </tr>
+  <tr>
+    <td>foodbank_name</td>
+    <td>Required</td> 
+    <td>Name of food bank</td>
+    <td>string</td>
+  </tr>
+</table>
+
+<h4 id="Response">Response</h4>
+
+<table style="width:100%">
+  <tr>
+    <th>Field</th>
+    <th>Description</th> 
+    <th>Data type</th>
+    <th>Example</th>
+  </tr>
+  <tr>
+    <td>foodbank_name</td>
+    <td>Name of food bank</td> 
+    <td>String</td>
+    <td>"Mid Norfolk Foodbank"</td>
+  </tr>
+   <tr>
+    <td>Items_needed</td>
+    <td>List of food items needed for specified food bank</td> 
+    <td>array [string]</td>
+    <td>"tinned fruit", "tinned rice pudding"</td>
+  </tr>
+</table>
+
+<h4 id="Input error">Input error</h4>
+<p>If the food bank doesn't exist, it will return the following string:
```

Comment:
Maybe this is the better place to add the note that it has to be an exact string match